packages/babel-plugin-popmotion/jest.config.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  transform: {
```

Comment:
because this plugin tests against both babel@6 and babel@7 I was more comfortable with providing the babel transform for jest explicitly instead of relaying on its automatic behaviour